src/ClusterManager/test/test_distributed_job.py

```diff
@@ -543,3 +543,28 @@ def test_fault_tolerance(args):
                                            cmd)
         assert code == 0, "code is %s, output is %s" % (code, output)
         assert output == "dummy\n", "output is %s" % (output)
+
+
+@utils.case()
+def test_image_pull_msg(args):
+    expected = "ImagePullBackOff"
```

Comment:
We should fail this case at some point. Maybe in a separate PR.

Reply:
Yes, this case is very hard to handle, because  when an image is huge, it will show this state until image is pulled successfully. We can have a timeout on this, but may fail on case of huge image.